packages/l10n/index.js

```diff
@@ -0,0 +1,13 @@
+export { countriesShape, withCountries } from './country-information';
+export { withCurrencies } from './currency-information';
+export { languagesShape, withLanguages } from './language-information';
+export { timeZonesShape, withTimeZones } from './time-zone-information';
+export {
+  default as extractLanguageFromLocale,
+} from './utils/extract-language-from-locale';
+export {
+  default as getSupportedLanguage,
+} from './utils/get-supported-language';
+export {
+  default as getSymbolFromCurrency,
+} from './utils/get-symbol-from-currency';
```

Comment:
I like that we generally move to a root export. Has been easier to work with as the location of a module inside of a package should be implementation detail to it. Ultimately it means less breaking changes.